apps/client/src/app/modules/freecompany-workshops/model/freecompany-airships.ts

```diff
@@ -0,0 +1,7 @@
+import { SectorExploration } from './sector-exploration';
+import { Airship } from './airship';
+
+export interface FreecompanyAirships {
+  sectors: { [id: string]: SectorExploration };
```

Comment:
I find `Record<K, V>` easier to read so i'm trying to use it whenever it's possible ^^

```suggestion
  sectors: Record<string, SectorExploration>;
```

Reply:
TIL 😄